apps/src/code-studio/header.js

```diff
@@ -108,10 +112,6 @@ header.build = function (scriptData, stageData, progressData, currentLevelId, pu
     e.stopPropagation();
     $('.header_popup').is(':visible') ? hideHeaderPopup() : showHeaderPopup();
   });
-  $('.header_popup').click(function (e) {
-    e.stopPropagation(); // Clicks inside the popup shouldn't close it
-  });
```

Comment:
In an ideal world, a lot of the logic here gets moved into a React component. I ultimately decided that was out of scope for now.